src/ProjectContents/ModalNewContent.tsx

```diff
@@ -0,0 +1,176 @@
+import { useEffect, useMemo, useState } from "react";
+import { Navigate } from "react-router-dom";
+import dayjs from "dayjs";
+import jsonSchemaToJaamSchema from "../@packages/json-schema-to-jaam-schema";
+import jsonSchemaValidator from "../@packages/json-schema-validator";
+
+import { SelectBox, TextField, useModal } from "../@shared";
+import { FieldTitle } from "../ProjectSchema/FieldTitle";
+import { useContentsState, useSetContentsState } from "./useContentsStore";
+import { useCreateContentMutation } from "./useContentsMutation";
+import * as css from "../ProjectSchema/ModalNewSchema.css";
+
+import type { SchemaList } from "../@types/api";
+import type { SchemaPropertyType } from "../@packages/json-schema-to-jaam-schema/types";
+
+type ModalNewContentProps = {
+  token: string;
+  schemaList?: SchemaList[];
+};
+
+type EditorType = "textfield" | "textarea" | "dateInput";
+
+const editorForType: Record<SchemaPropertyType, EditorType> = {
```

Comment:
```suggestion
const EDITOR_BY_TYPE: Record<SchemaPropertyType, EditorType> = {
```

상수는 대문자로 선언되어야 할 것 같아요~

Reply:
스키마 수정하면서 함께 수정해볼게요~